modules/insights-operator-advisor-recommendations.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * support/remote_health_monitoring/using-insights-to-identify-issues-with-your-cluster.adoc
+
+:_content-type: CONCEPT
+[id="insights-operator-advisor-recommendations_{context}"]
+= Understanding Insights Advisor recommendations
+
+Insights Advisor bundles information about known configuration issues that can negatively affect the service availability, fault tolerance, performance, or security of your clusters. This information set is called a recommendation in Insights Advisor and includes the following information:
```

Comment:
s/ known configuration issues/various clusters states and component configuration